config/locales/en/divisions.en.yml

```diff
@@ -0,0 +1,13 @@
+en:
+  # Note, 'New Division' needs to be shared between the 'index' and 'new' actions, so can't use
+  # ".new" shortcut, and 'admin.divisions.new' seems too long.
+  # Todo, Confirm how exactly to name these kinds of shared keys.
```

Comment:
I would say `divisions.new_division` and `divisions.parent` are fine, just perhaps put a comment above the `admin` key saying that it's there because rails requires it for using the shortcut, so below are template-specific translations.